### DISCUSSION TITLE
How to round the ends of a gauge bar?

### DISCUSSION THREAD

**Discussion by WebGoose** [Live-Charts/LiveCharts2#488] [Q&A]
Hi, 

In the examples gif that plays there is the following shot: 

![image](https://user-images.githubusercontent.com/5024960/169764103-f764d4a8-e0c3-4940-a804-4ea35f411ffa.png)

How do I round the ends of the series like that? I Can't seem to find it in the documentation

**Comment by beto-rodriguez** [comment]
Hi

This is an experimental feature that should be available soon.
